src/pybind/mgr/progress/module.py

```diff
@@ -624,10 +639,13 @@ def update(self, ev_id, ev_msg, ev_progress, refs=None):
         """
         For calling from other mgr modules
         """
+        if not self.enabled:
+            self.log.info("remote update failed, progress module not enabled")
```

Comment:
these logs should be debug level - it's more useful for devs than users to know that progress is not being tracked when the module is disabled. it'd become too frequent in user-level logs.